app/src/main/java/org/eyeseetea/malariacare/domain/exception/NetworkException.java

```diff
@@ -0,0 +1,8 @@
+package org.eyeseetea.malariacare.domain.exception;
+
+
+public class NetworkException extends Exception {
+    public NetworkException() {
+        super("Credentials not valid");
```

Comment:
This string should not be changed, it is an exception message that is launched from domain, Domain can not translate, this must be done only in activities and fragments. Next branch is translated in the right place